src/js/steps/letters/generateCharPicker.js

```diff
@@ -86,7 +86,7 @@ module.exports = function (options, monkeyContainer) {
           $toolTip.appendTo($pickerContainer);
           var $closeButton = $('<button>')
               .attr('type', 'button')
-              .addClass('button primary character-picker__close')
+              .addClass('button button--primary character-picker__close')
```

Comment:
Nope, we actually had to put a stopgap in for the `primary` class when we did the BEMification stuff. Will be absolutely fine now.